notebooks/geo/geoseries.ipynb

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
<p>Same here. Add another cell after this one.</p><p> </p><blockquote><code>point_geom_series.geo.y</code></blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/1327/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>